src/emulator/emulatorLogger.ts

```diff
@@ -84,7 +84,7 @@ export class EmulatorLogger {
         EmulatorLogger.log("USER", `${clc.blackBright("> ")} ${log.text}`);
         break;
       case "DEBUG":
-        if (log.data && log.data !== {}) {
+        if (log.data && Object.keys(log.data).length > 0) {
```

Comment:
In ES5, if the argument to this method is not an object (a primitive), then it will cause a TypeError. In ES2015, a non-object argument will be coerced to an object.